src/odemis/driver/smarpod.py

```diff
@@ -870,3 +1117,568 @@ def Smarpod_GetDLLVersion(self, p_major, p_minor, p_update):
         minor.value = 2
         update = _deref(p_update, c_uint)
         update.value = 3
+
+
+class SmarAct_MC(model.Actuator):
```

Comment:
Let's call this class "MCTriGlide".

Reply:
Apparently, it's actually not a triglide. Any alternative ideas for the name?